package.json

```diff
@@ -14,21 +14,370 @@
     "url": "https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1/issues"
   },
   "jest": {
-    "setupFilesAfterEnv": ["jest-extended"],
+    "setupFilesAfterEnv": [
+      "jest-extended"
+    ],
     "projects": [
       {
         "displayName": "mandatory",
-        "testMatch": ["<rootDir>/mandatory/*.js"]
+        "testMatch": [
+          "<rootDir>/mandatory/*.js"
+        ]
       },
       {
         "displayName": "extra",
-        "testMatch": ["<rootDir>/extra/*.js"]
+        "testMatch": [
+          "<rootDir>/extra/*.js"
+        ]
       }
     ]
   },
   "homepage": "https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1#readme",
   "devDependencies": {
     "jest": "^26.6.3",
     "jest-extended": "^0.11.5"
-  }
+  },
+  "main": "index.js",
```

Comment:
something whacky definitely occurred here with your npm; hopefully the discussions we've had will help resolve things